### ISSUE TITLE
Slider field triggers javascript errors in backoffice on certain settings

### ISSUE THREAD

**Issue by karlmacklin** [umbraco/Umbraco-CMS#13029]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.2.0

### Bug summary

Certain settings on the slider field triggers some form of error in the backoffice stack.
The javascript error triggered is:

```
angular.js:15697 TypeError: Cannot read properties of null (reading 'classList')
    at Object.<anonymous> (umbraco.directives.min.js?d=637991740050000000:1:400753)
    at nouislider.min.js?umb__rnd=0987c79e3c15fd6fee7b2f53c5f008fd3a7ecb45:1:21100
    at Array.forEach (<anonymous>)
    at nouislider.min.js?umb__rnd=0987c79e3c15fd6fee7b2f53c5f008fd3a7ecb45:1:21078
    at Array.forEach (<anonymous>)
    at I (nouislider.min.js?umb__rnd=0987c79e3c15fd6fee7b2f53c5f008fd3a7ecb45:1:21024)
    at nouislider.min.js?umb__rnd=0987c79e3c15fd6fee7b2f53c5f008fd3a7ecb45:1:20750
    at Array.forEach (<anonymous>)
    at Object.X [as on] (nouislider.min.js?umb__rnd=0987c79e3c15fd6fee7b2f53c5f008fd3a7ecb45:1:20728)
    at setUpActivePipsHandling (umbraco.directives.min.js?d=637991740050000000:1:400472) 'Possibly unhandled rejection: {}'
```

It happens whenever you have set up a slider field that has a default value of 0,5 while max allowed value is 0.
However it can also happen when you have an increment set to 0,1 and the rest of the settings are "logical" (compared to have a default value higher than 0 while max allowed is 0).
For example this will trigger the error:

![image](https://user-images.githubusercontent.com/353806/190981941-b9d2217a-8aeb-47b0-9b7f-687c541455e3.png)

~~**This bug is critical** in the sense that it "kills" the "picture in picture" slideout editor for certain field types. For example the form picker in Umbraco Forms is not possible to open at all while this error is triggered.~~

### Steps to reproduce

- Fresh install Umbraco 10.2.0.
- Create new document type with a tab and field group.
- Add a slider field to the group.
- Configure the slider to have `0,5` as a default value.
- Save the document type and reload the page.
- Check javascript console.

~~Also if you have Umbraco Forms installed, you can try adding a form picker field to the same page and try to open the picker. In that case the picker is never shown.~~

### Expected result / actual result

No javascript error.

---
_This item has been added to our backlog [AB#22651](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/22651)_

**Comment by karlmacklin** [comment]
After more testing and investigating we have found cases where this does not affect the form picker, so that part is now irrelevant/unconfirmed.

**Comment by nul800sebastiaan** [comment]
At the moment, I can't reproduce any case where any panels are being prevented from sliding in due to the error we see in the console, but I can indeed confirm there's a console error when the range slider has a default value other than 0, I believe it also required the step value to be not 0, not sure. 

As you say, even a form picker should work:
https://user-images.githubusercontent.com/304656/191001589-6cbca348-aa20-4c78-9f21-491df8867727.mp4

However, the problem you're seeing very much sounds like the problem we fixed in Forms 10.1.2, so I would recommend checking that you're very sure that you've updated to Forms 10.1.2. Maybe start with completely deleting the `bin` AND `obj` folders and do a build + run of the project after making sure that Forms is of version 10.1.2 in your csproj file.

If you have multiple csproj files in your solution, they should all have the correct forms reference in them.

**Comment by karlmacklin** [comment]
Right you are sir, not sure where in our testing process we messed up but we did mix up the versions of Umbraco Forms between our testing, and clearing bin and obj and restarting everything on the right version seems to have eradicated the critical problem.

I guess this issue can remain for those weird slider issues?
I'll edit the original post to not signal any criticality.

**Comment by nul800sebastiaan** [comment]
Excellent news, yes we're working on the slider soon as that seems a regression (worked without error in 10.1.1).

**Comment by bjarnef** [comment]
It is related to this https://github.com/umbraco/Umbraco-CMS/pull/12807 as the log error mention `setUpActivePipsHandling`.

It doesn't check the element exists when adding the class, only when removing the class.

**Comment by iOvergaard** [comment]
Hello everyone and thanks for your reports and testing. This will be fixed in an upcoming patch to 10.2.1 so standby for that!

**Comment by nikolajlauridsen** [comment]
Fixed in #13044
